frontend/src/features/properties/list/interfaces.ts

```diff
@@ -41,9 +41,9 @@ export interface IProperty {
   // Parcel Properties
   landArea: number;
   landLegalDescription: string;
-  municipality: string;
-  zoning: string;
-  zoningPotential: string;
+  municipality?: string;
+  zoning?: string;
+  zoningPotential?: string;
```

Comment:
These are not being returned by the API currently so had to make them optional. If that goes against business rules, the API needs to return empty or null properties in the JSON response

Reply:
Currently the API does not return null valued properties